### PR TITLE
Adding updateEmployer. 

### PR DESCRIPTION
**Pull request by Tsaikkari**
Test issue.

### CODE REVIEW

File: src/routers/employer.ts [hunk 1]
```diff
@@ -11,5 +13,7 @@ const router = express.Router()
 router.post('/login/local', localLogin)
 router.post('/create', registerEmployer)
 router.post('/jobs/:companyName', createJobPost)
+router.get('/', getEmployer)
```
Comment (Wyfy0107) [inline src/routers/employer.ts:16]: You have to provide an employer id as a param to get that specific employer

File: src/controllers/employer.ts [hunk 2]
```diff
@@ -65,6 +65,58 @@ export const registerEmployer = async (
...
 }
 
+//**Get all employers*/
+export const getEmployer = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const user = await Employer.find({ relations: ['credentials'] })
```
Comment (Wyfy0107) [inline src/controllers/employer.ts:75]: Extract employer id from params and use it to find that employer

File: test/controllers/employer.test.ts [hunk 3]
```diff
@@ -96,6 +96,36 @@ describe('user controller', () => {
     expect(response.body.id).toBe(1)
   })
 
+  it('should get employers', async () => {
+    await createEmployer()
+    const response = await request(app)
+      .get('/employer')
```
Comment (Wyfy0107) [inline test/controllers/employer.test.ts:102]: provide the employer id as params

File: test/controllers/employer.test.ts [hunk 4]
```diff
@@ -96,6 +96,36 @@ describe('user controller', () => {
...
+
+    expect(response.status).toBe(200)
+  })
+
+  it('should update employer', async () => {
+    await createEmployer()
+    const employerId = await request(app)
+      .get('/employer/:id')
```
Comment (Wyfy0107) [inline test/controllers/employer.test.ts:110]: The param should be a number: id of that newly created employer. Ex: `.get('/employer/1)`

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

File: src/routers/employer.ts (hunk 1 above)
Comment (Tsaikkari) [reply src/routers/employer.ts:16]: That's the router for getting all the employers.

File: test/controllers/employer.test.ts (hunk 3 above)
Comment (Tsaikkari) [reply test/controllers/employer.test.ts:102]: This test was not failing.

File: src/routers/employer.ts (hunk 1 above)
Comment (Tsaikkari) [reply src/routers/employer.ts:16]: I change the name to getEmployers

File: test/controllers/employer.test.ts (hunk 4 above)
Comment (Tsaikkari) [reply test/controllers/employer.test.ts:110]: Thanks. I changed it to a number, but the test is still failing.

**Pull request closed without merging**